pkg/controller/provider/web/base/client.go

```diff
@@ -307,6 +307,11 @@ func (c *RestClient) buildTransport() (err error) {
 		transport.TLSClientConfig = &tls.Config{
 			RootCAs: pool,
 		}
+	} else {
```

Comment:
how do we know we are in dev mode here, should we be more explicit to to be more clear we don't want this case triggered on production ?